src/coreclr/tools/InjectResource/Program.cs

```diff
@@ -1,29 +1,21 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
+
 using System.CommandLine;
 using System.IO;
 
-var binOption = new Option<FileInfo>(
-    name: "--bin",
-    description: "Binary data to attach to the image");
-var imageOption = new Option<FileInfo>(
-    name: "--image",
-    description: "PE image to add the binary resource into");
-var nameOption = new Option<string>(
-    name: "--name",
-    description: "Resource name");
-var rootCommand = new RootCommand("Inject native resources into a Portable Executable image");
-rootCommand.AddOption(binOption);
-rootCommand.AddOption(imageOption);
-rootCommand.AddOption(nameOption);
+CliOption<FileInfo> binOption = new("--bin") { Description = "Binary data to attach to the image" };
+CliOption<FileInfo> imageOption = new("--image") { Description = "PE image to add the binary resource into" };
+CliOption<string> nameOption = new("--name") { Description = "Resource name" };
+var rootCommand = new CliRootCommand("Inject native resources into a Portable Executable image");
```

Comment:
nit: style consistency with other symbols ctor usage

```suggestion
CliRootCommand rootCommand = new("Inject native resources into a Portable Executable image");
```